modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/AccountHeader.kt

```diff
@@ -160,18 +162,6 @@ internal sealed interface SubscriptionHeaderState {
         val platform: SubscriptionPlatform,
         val giftDaysLeft: Int,
     ) : SubscriptionHeaderState
-
-    data class SupporterRenew(
-        override val tier: SubscriptionTier,
-        override val expiresIn: Duration?,
-        override val isChampion: Boolean,
-    ) : SubscriptionHeaderState
-
-    data class SupporterCancel(
-        override val tier: SubscriptionTier,
-        override val expiresIn: Duration?,
-        override val isChampion: Boolean,
-    ) : SubscriptionHeaderState
```

Comment:
These 2 types are not needed. They were used when the old `SubscriptionTier` was `NONE`. But it is not possible. Only `Patron` and `Plus` are available in [the server's code](https://github.com/Automattic/pocket-casts-sync-api/blob/8227f70e22a1416c95f662afb8e2a8476cb8b446/api/modules/subscriptions/src/main/kotlin/com/pocketcasts/subscriptions/data/Subscription.kt#L39-L42).